setup.py

```diff
@@ -1,9 +1,20 @@
 from setuptools import setup, find_packages
 
+# We define the version number in restructuredtext_lint/__init__.py
+# Here we can't use "import restructuredtext_lint" to access it
+# as "restructuredtext_lint.__version__" as that would tell us the
+# previously installed version (if any).
```

Comment:
Verbose is fine, but it can still be done in 2 lines; 4 is overkill. Here would be my variant:

```python
# Retrieve `__version__` from `__init__.py`
# DEV: We load `__version__` via regexp matching to avoid loading `rst-lint` and its dependencies
```